.github/workflows/vs2019/run-tests.bat

```diff
@@ -0,0 +1,12 @@
+@REM Run tests on visual studio 2019 generated build
```

Comment:
I don’t like putting the logic into bat files at all, this makes it really hard to see what's going on when looking at the job on github.

This isn't practically reusable anyway because there are so many details in here that you'd want to do different in different jobs. Strong preference for job logic to be all contained in the job script unless it's either doing an actually usefully reusable thing or is so long/complicated that it doesn't reasonably fit there.

Reply:
I have to disagree here somewhat. I like the `.bat` file for this. Having the logic in `.bat` files makes it much easier to try out locally. Otherwise more copy and pasting from the `.yml` file is required.

Reply:
I'm somewhere in the middle with this - I see Hannes' point, and I actually prefer that - but for consistency's sake with the current infrastructure, I'd rather it stays a batch script.

I could put it on my todo if people insist to port both of these to the github action workflow file, but I would much rather this happens at a later PR.